gitfiti.py

```diff
@@ -168,6 +170,41 @@ def load_images(img_names):
         loaded_imgs[name] = json.loads(img_list)
         return loaded_imgs
 
+
+max_size = 40, 7
+
+def from_image(fin):
```

Comment:
To make PIL a soft requirement, move the import inside the function where you need it(i.e. here), and do something like this:

    try:
        import PIL
    except ImportError:
        print 'Error: No imaging library is installed'
        print 'Run "pip install Pillow" before using this functionality'
        sys.exit(1)